frontend/src/pages/Home.jsx

```diff
@@ -55,14 +127,14 @@ const Home = () => {
   const statusStyle = getStatusStyle();
 
   const handleReqIdChange = (event) => {
```

Comment:
@Hoshi-Okami Please rename it to handleSimulationTypeChange() to better match the context